src/docker.rs

```diff
@@ -126,6 +136,7 @@ pub fn run(
     let runner = config.runner(target)?;
 
     let mut docker = docker_command("run")?;
+    let is_docker = get_is_docker(get_container_engine().unwrap(), verbose)?;
```

Comment:
Note that this can't fail due to the above command: `docker_command` ensures that `get_container_engine` is not an error.